bitbucket/pull_requests.go

```diff
@@ -36,8 +36,10 @@ type PullRequest struct {
 	Reason            *string            `json:"reason,omitempty"`
 	UpdatedOn         *string            `json:"updated_on,omitempty"`
 	Author            *User              `json:"author,omitempty"`
-	MergeCommit       *string            `json:"merge_commit,omitempty"`
-	ClosedBy          *User              `json:"closed_by,omitempty"`
+	MergeCommit       struct {
```

Comment:
Could we instead create another struct here called `PullRequestMergeCommit` instead of an inline struct definition? This should set us up later on if they add additional attributes to `merge_commit`. Please run `go run gen-accessors.go` afterwards.

Reply:
Thanks for your guidance, that makes sense. I will change it during the next 1,2 days.

Reply:
I have added the struct; the tests run successfully.